README.md

```diff
@@ -45,6 +50,7 @@ Each state's crawler is put under its own folder (e.g. `states/new_york/main.py`
   ```make test```
 
 - Also, rerun the Jupyter notebook `analysis/Analysis.ipynb` from scratch to update the analytics.  You can see how many fields you were able to parse.  To start the jupyter notebook, run `make jupyter`.  Run the notebook.  Make sure that you have all the values you need.  **Do not commit the notebook changes**.  Jsut throw them away.  They just block rebase merging.
+- To release a new data version between scheduled run dates, ***first make sure the data file passes all tests*** using `inv test`. Then, commit the passing data file to the [public-data branch](https://github.com/mail-my-ballot/elections-officials/tree/public-data) and tag that commit with a date version of format `data/2020-06-22`.
```

Comment:
I'm a bit wary of adding an inv task that does that many git actions. It would involve:

1. running a state
2. running test
3. copying the .json file somewhere
4. switching branches to public-data
5. fetch/pull
6. copy .json file back to public
7. add .json file
8. commit (if different)
9. tag (if different)

I think there are too many things that could go wrong and inadvertently really mess up someone's local repo.

Perhaps we could come up with an additional trigger to just run the public_data workflow on demand in those situations?

Reply:
Perhaps an inv task that submits a POST request as an [external repository dispatch](https://help.github.com/en/actions/reference/events-that-trigger-workflows#external-events-repository_dispatch) to the public_html workflow? Or to a new workflow if we want it to be state-specific and pass the state as part of the [client payload](https://developer.github.com/v3/repos/#create-a-repository-dispatch-event)?

Seems like a lot of effort for not much gain, but it is possible.

Reply:
OK -- let's merge for now and discuss this somewhere else.